src/components/cart/CartToggle/cart-toggle.scss

```diff
@@ -0,0 +1,28 @@
+.cart-toggle[data-store-button] {
+  position: relative;
+  background-color: transparent;
+  border-width: 0;
+  cursor: pointer;
+
+  &::after {
+    position: absolute;
+    top: var(--space-0);
+    left: var(--space-4);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: var(--space-4);
+    padding: 1px var(--space-0);
+    color: var(--color-text-negative);
+    font-weight: var(--text-weight-bold);
+    font-size: var(--text-size-0);
+    background: var(--bg-secondary-default);
+    border-radius: var(--border-radius-pill);
+    content: attr(data-items);
+  }
+
+  img {
```

Comment:
Not sure, but do we have img on this component? 👀

Reply:
We haven't! I forgot to remove this.